18F/identity-idp
# Remove request_id parameters from Sign In view links

## 🛠 Summary of changes

Removes `request_id` parameter assignment to links rendered in the Sign In page.

It's seemingly unnecessary to include the `request_id` in these links, since the value would have already been persisted to the session by this point ([source](https://github.com/18F/identity-idp/blob/2136e1ff144cafce60cdde6ca468042dde54f99c/app/controllers/users/sessions_controller.rb#L15)). It seems that this might be a hold-over from an initial implementation which didn't persist `request_id` to the session, as the session persistence was introduced in #1275, at which time [the view links already included the parameter](https://github.com/18F/identity-idp/blob/15b624fedddbf6436a13229717d74a798f09ba34/app/views/devise/sessions/new.html.slim#L20).

Related discussion: https://github.com/18F/identity-idp/pull/8112/files#r1154794719

## 📜 Testing Plan

1. Start [OIDC Sample App](https://github.com/18f/identity-oidc-sinatra) in a separate process.
2. Go to http://localhost:9292
3. Click Sign in
4. Click "Create an account" or "Forgot password?" links
5. Observe SP logo (18F) is still in header
